test/README.md

```diff
@@ -0,0 +1,100 @@
+# Keycloak JS test suite
+
+This directory contains the test suite for Keycloak JS, which is based on [Playwright](https://playwright.dev/). It contains a suite of integration tests that embed the adapter in various scenarios and tests it against a Keycloak server running in the background.
+
+## Setup
+
+Run the following command to install the [Playwright browsers](https://playwright.dev/docs/browsers) and required system dependencies:
+
+```sh
+npx playwright install
+```
+
+It might be that this command fails due to missing system dependencies, in that case add the `--with-deps` flag:
+
+```sh
+npx playwright install --with-deps
+```
+
+### Setup on unsupported Linux distributions
```

Comment:
Maybe this should read: Setup on Linux distributions unsupported by Playwright

Reply:
I've added some details that this section can be skipped if you are not on an unsupported Linux distro.